ui/pages/confirmation/confirmation.js

```diff
@@ -176,6 +179,9 @@ export default function ConfirmationPage({
     isEqual,
   );
   const unapprovedTxsCount = useSelector(getUnapprovedTxCount);
+  const approvalFlows = useSelector(getApprovalFlows, isEqual);
+  const hasPendingApprovals = useSelector(hasPendingApprovalsSelector);
```

Comment:
Does this selector not require at least an `approvalType` argument meaning this will always return an empty array as no approvals have the type `undefined`?

Do we need an alternate more general selector?

Reply:
Wrong selector. I've changed it.

I think we might want to consider adding some sort of tests to this confirmation page in the nearby future.